### PR TITLE
Update sites.yml

### PR DESCRIPTION
**Pull request by EdenTamrat**
Hello, I am looking add my update site onto the sites.yml list.

### CODE REVIEW

File: sites.yml [hunk 1]
```diff
@@ -1792,6 +1792,16 @@ sites:
     maintainers:
       - "[Niko Ehrenfeuchter](https://imagej.net/people/ehrenfeu)"
 
+  - name: "Nscale_segmentaion"
+    id: "Nscale_segmentaion"
+    url: "https://sites.imagej.net/Nscale_segmentaion/"
+    description: >-
+      A customizable segmentation tool for fluorescence cell microscopy 
+      analysis used to separate clustered/touching nuclei. For more information- 
+      https://github.com/EdenTamrat/imagej.github.io/blob/1069c0beda8e4f0d5bd70e055b5a294446efca6b/_pages/plugins/Nscale-segmentation.md
```
Comment (ctrueden) [inline sites.yml:1801]: This URL should be `https://imagej.net/plugins/Nscale-segmentation`, and it should be made into a hyperlink with Markdown syntax.

**Review by ctrueden — CHANGES_REQUESTED** [review]

**Comment by ctrueden** [comment]
Thanks @EdenTamrat. I will fix my comment above post-merge, so don't worry about fixing it yourself. :smile:

**Comment by ctrueden** [comment]
Actually... before I merge this: @EdenTamrat I noticed that the name of the update site is `Nscale_segmentaion` rather than `Nscale-segmentation`. Shall we rename the update site to fix this typo, before adding it to the advertised list? Or is that spelling intentional?

**Comment by EdenTamrat** [comment]
Hello, thank you for your comments. 
The typo was unintentional; I meant to write Nscale_segmentation like you mentioned. 
I would like to rename my update site to fix the error, if its possible.

**Comment by hinerm** [comment]
@EdenTamrat I renamed the [update site](https://sites.imagej.net/Nscale_segmentation/) and then updated your PR. For reasons I don't understand I was able to update your PR once but it updated to the current `main` instead of the corrected PR. So I just pushed the changes. Apologies for blowing away your PR. If you need anything else though please let me know.

**Comment by ctrueden** [comment]
Thanks @hinerm for dealing with this, and sorry @EdenTamrat for dropping the ball on it for so long. Please let us know if you encounter any issues with using the update site!

**Pull request closed without merging**